src/sentry/event_manager.py

```diff
@@ -2056,13 +2055,19 @@ def _get_severity_score(event: Event) -> float | None:
     # to BigQuery for storage space reasons (stacktraces can be enormous). Since the ML model is
     # trained on BQ data, we have to use the title to match.
     #
-    # TODO: Figure out if there's a way to get the full error message to the model.
+    # TODO: Figure out if there's a way to get the full error message to the model. (If we do that,
+    # though, it'll only work for `ErrorEvent`-type events. We'll still have to use `title` for
+    # `DefaultEvent`-type events - like those which come from `capture_message` calls - or find the
+    # message data elsewhere in the event.)
     title = event.title
+    event_type = get_event_type(event.data)
     if title in NON_TITLE_EVENT_TITLES:
-        title = ErrorEvent().compute_title(dict(event.get_event_metadata()))
+        title = event_type.get_title(event_type.get_metadata(event.data))
```

Comment:
Maybe I'm missing something - is this not the same as what `event.title` does?

https://github.com/getsentry/sentry/blob/16d2fbd110c52297a9f0d7ac81d9621430a8f62b/src/sentry/eventstore/models.py#L255-L256

Reply:
In my testing, there were some events which came through to that point in the process with `<untitled>` as their `event.title`, so this just forces the title computation to run if for some reason it hasn't yet. If we still get garbage back, we don't send the request.

I can add a comment.